plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryNamedRelationHandle.java

```diff
@@ -78,6 +81,12 @@ public Optional<String> getComment()
         return comment;
     }
 
+    @JsonProperty
+    public boolean hasConnection()
+    {
+        return connectionId.isPresent();
```

Comment:
There is no need to keep `Optional<String>` as a field if we are interested only in boolean.